bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTOMapper.java

```diff
@@ -1,11 +1,10 @@
 /**
- * Copyright (c) 2016 Markus Rathgeb
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
```

Comment:
We already agreed that "and others" has been missing. I haven't forget that and yes, it has been merged to early before I have had any change to add them.

Perhaps my reading of https://eclipse.org/legal/copyrightandlicensenotice.php is wrong.
The copyright owner differs from author / committer, but it fits to the one that initially contributed that file ("is the copyright owner that created the initial content").

```
A copyright owner may be either an individual or an entity. 
```
* an individual is allowed